core-web/libs/dot-primeng-theme-styles/src/scss/_variables.scss

```diff
@@ -139,6 +139,7 @@ $font-size-small: 0.85rem;
 $font-size-x-small: 0.75rem;
 $font-weight-regular-bold: 400;
 $font-weight-semi-bold: 500;
```

Comment:
```suggestion

```

Reply:
isn't this used on other places? meaning it can not be removed without changing the var name  on other places